build/manifest/main.go

```diff
@@ -82,7 +82,7 @@ func findManifest() (*model.Manifest, error) {
 	// we don't want to accidentally clobber anything we won't preserve.
 	var manifest model.Manifest
 	decoder := json.NewDecoder(manifestFile)
-	decoder.DisallowUnknownFields()
```

Comment:
This is commented out to support compiling the plugin with an "unknown" property on the plugin settings. Updating the server version to otherwise support this is not trivial because:

- The newer mattermost-server commit to import requires Go 1.18 to compile
- circleci then needs to be updated to use Go 1.18
- Updating circleci's Go image is blocked by https://github.com/mattermost/circleci-orbs/pull/29
- I tried to continue the work on that PR, but getting Go 1.18 working on there was non-trivial
- The `golang:1.18-node` image also updates the node version, making it so it is a newer version than this plugin currently compiles with

Reply:
cc @hanzei for discussion on this

Reply:
Details on the error I was getting in CI https://github.com/mattermost/circleci-orbs/pull/29#issuecomment-1192221776